lambdas/db-migration/src/migrations/20201014105058_create_granules_table.ts

```diff
@@ -73,6 +73,12 @@ export const up = async (knex: Knex): Promise<void> =>
       .comment('Timestamp for granule production date/time');
     table
       .timestamp('timestamp');
+    table
+      .text('data_type')
+      .comment('Refers to a collection name');
+    table
+      .text('version')
+      .comment('Refers to a collection version');
```

Comment:
After looking closer at the code which writes records to Dynamo, it's not setting `data_type` and `version`:

https://github.com/nasa/cumulus/blob/feature-rds/packages/api/models/granules.js#L381

So these values are never making it into the Dynamo table, as far as I can tell. Thus, I'm not sure if we need to save them for Postgres

Reply:
Thanks for noticing that. I don't think we need to save them now as well. Will update accordingly